### PR TITLE
Skip the daemon's name when detecting MPRIS2 players.

### PR DESCRIPTION
**Pull request by lenoch**
The daemon's name ("org.mpris.MediaPlayer2.osdlyrics", defined as APP_MPRIS2_NAME in daemon/main.py) has the same prefix as MPRIS2 players.

It makes sense to skip it so we do not wait at the start if no player is actually running.

### CODE REVIEW

**Comment by nicman23** [comment]
was about to open a new issue :P

**Comment by lenoch** [comment]
Haha :-) We discussed it yesterday at https://gitter.im/OSDLyrics/Lobby
You can test this if you want; I will send a more polished version of the patch later.

**Comment by nicman23** [comment]
I already have and it works. Do not know about polish as I am not a programmer :P

File: players/mpris2/mpris2.py [hunk 1]
```diff
@@ -58,8 +58,10 @@ def _get_player_from_bus_names(self, names):
         Arguments:
         - `names`: list of bus names
         """
-        return [player_info_from_name(name[len(MPRIS2_PREFIX):]) for name in names
-                if name.startswith(MPRIS2_PREFIX)]
+        return [player_info_from_name(name[len(MPRIS2_PREFIX):])
+                for name in names if name.startswith(MPRIS2_PREFIX) and
+                # skip self: APP_MPRIS2_NAME (in daemon/main.py)
```
Comment (Corax26) [inline players/mpris2/mpris2.py]: Couldn't we move that constant to `python/consts.py`?  This way we could filter out `APP_MPRIS2_NAME` explicitly.

Actually there are three constants in `daemon/main.py` that would probably be better found in `python/consts.py`, that could be done in this commit or a separate one.

Comment (lenoch) [inline players/mpris2/mpris2.py]: Done in a separate commit.
